pkg/util/virt-chroot.go

```diff
@@ -0,0 +1,42 @@
+package util
+
+import "os/exec"
+
+const (
+	binaryPath     = "/usr/bin/virt-chroot"
+	mountNamespace = "/proc/1/ns/mnt"
+)
+
+func getBaseArgs() []string {
+	return []string{"--mount", mountNamespace}
+}
+
+func GetChrootBinaryPath() string {
+	return binaryPath
+}
+
+func GetChrootMountNamespace() string {
+	return mountNamespace
+}
+
+func MountChRoot(sourcePath, targetPath string, ro bool) *exec.Cmd {
+	args := append(getBaseArgs(), "mount", "-o")
+	optionArgs := "bind"
+
+	if ro {
+		optionArgs = "ro," + optionArgs
+	}
+
+	args = append(args, optionArgs, sourcePath, targetPath)
+	return exec.Command(binaryPath, args...)
+}
+
+func UnmountChRoot(path string) *exec.Cmd {
+	args := append(getBaseArgs(), "unmount", path)
```

Comment:
@iholder-redhat crucial typo. `unmount` -> `umount`

Reply:
@iholder-redhat perhaps it's worth to change function name as well. `UnmountChRoot` -> `UmountChRoot`

Reply:
You're my hero! Thanks!